automatic/7zip.commandline/7zip.commandline.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>7zip.commandline</id>
+    <title>7-Zip (Portable, CommandLine)</title>
+    <version>0.0</version>
+    <authors>Igor Pavlov</authors>
+    <owners>Rob Reynolds, chocolatey</owners>
```

Comment:
ah thanks, I forgot about that one.
Don't know why it's in there though, as the owners element isn't used on chocolatey.org as far as I know.